<launch>

<node pkg="rqt_multiplot" name="rqt_multiplot" type="rqt_multiplot" output="screen"/>

<node pkg="nimo_end_effector" name="nimo_end_effector" type="nitrate_sample.py" output="screen"/>

</launch>
